nodes.lua

```diff
@@ -1,5 +1,5 @@
 local function isempty(s)
-  return s == nil or s == ''
+  return s == nil or s == '' or 0 == s
```

Comment:
trival ` or 0 == s` should be ` or s == 0`

Reply:
I'd rather change the other two to the 'correct' order :p